src/content/docs/start/prerequisites.mdx

```diff
@@ -263,34 +263,33 @@ export JAVA_HOME="/Applications/Android Studio.app/Contents/jbr/Contents/Home"
 - Android SDK Build-Tools
 - Android SDK Command-line Tools
 
-4. Set `ANDROID_HOME` and `NDK_HOME` environment variables. Replace the version numbers so it matches your installed version.
+Selecting "Show Package Details" in the SDK Manager enables the installation of older package versions. Please only select other than the current packages if you know what you are doing.
 
-{/* TODO: Does the version number change below? */}
+4. Set `ANDROID_HOME` and `NDK_HOME` environment variables. Replace the version numbers so it matches your installed version.
```

Comment:
> Replace the version numbers so it matches your installed version.

For clarity, what version number am I supposed to replace?

Reply:
That's no longer neccessary, so I've removed that part.